samples/compare/src/test/kotlin/okhttp3/compare/JavaHttpClientTest.kt

```diff
@@ -15,30 +15,35 @@
  */
 package okhttp3.compare
 
+import mockwebserver3.MockResponse
+import mockwebserver3.MockWebServer
 import java.net.http.HttpClient
 import java.net.http.HttpClient.Redirect.NORMAL
 import java.net.http.HttpRequest
 import java.net.http.HttpResponse.BodyHandlers
-import mockwebserver3.MockResponse
-import mockwebserver3.MockWebServer
+import okhttp3.testing.PlatformRule
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
 
 /**
  * Java HTTP Client.
  *
  * https://openjdk.java.net/groups/net/httpclient/intro.html
+ *
+ * Baseline test if we ned to validate OkHttp behaviour against other popular clients.
  */
-@Disabled("Requires Java 11, but OkHttp runs on Java 8+")
```

Comment:
It's interesting that @Ignore was sufficient with JUnit 4 but @Disabled is not on JUnit 5.

Reply:
True, but this seems better anyway